packages/dependency-metatx/contracts/ERC2771HandlerAbstract.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: MIT
+// solhint-disable-next-line compiler-version
+pragma solidity ^0.8.18;
+
+/// @dev minimal ERC2771 handler to keep bytecode-size down
+/// based on: https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/metatx/ERC2771Context.sol
+abstract contract ERC2771HandlerAbstract {
```

Comment:
Context doesn't add anything and it force us add to add an extra external dependency (because of that we call it Handler). 
It is different for OZ lib, there it makes more sense.